### ISSUE TITLE
Set Visitors Name 

### ISSUE THREAD

**Issue by Manegoldsr21** [LivePersonInc/developers-community#707]
Hi,

I stumbled across this when trying to look if I could set the visitors name while using the Chat Agent API.

https://developers.liveperson.com/undefined-set-the-visitors-name.html

I was wondering if this is implemented yet? I am getting an error when I try to call it. I am able to successfully retrieve a visitor name but I need a way to update it.

https://developers.liveperson.com/chat-agent-api-methods-retrieve-visitor-s-typing-status.html

**Comment by mjansonius** [comment]
I found out just now that setting the Engagement Attributes personal info firstname and/or lastname will set the Visitor Name.
I don't think this is documented anywhere. Engagement Attributes API, called from FaaS, to set the firstname from a bot in Conversation Builder (tested on web messaging, by the way, didn't test on live chat). 
 I actually had only one field in the source data (fullname) so I just set the firstname and it showed up in the Visitor name (which I actually had not expected!).
